### PR TITLE
Update documentation to run UCS emulator.

### PR DESCRIPTION
**Pull request by mmnelemane**
Added a sample .xml file for running the emulator using KVM with additional steps for clarity.

### CODE REVIEW

File: crowbar_framework/public/docs/cisco_ucs.md [hunk 1]
```diff
@@ -75,10 +75,19 @@ All troubleshooting should be done within the Cisco UCS Manager interface.
 
 The UCS Platform Emulator can be [downloaded from Cisco's
 website](http://developer.cisco.com/web/unifiedcomputing/ucsemulatordownload)
-after registering for a free account.  This barclamp has been tested
-with version 2.1(2aPE1) of the emulator.  However at the time of writing, 
-the emulator does not honour power commands, so do not expect this 
-functionality to work.
+after registering for a free account.
+
+Steps to start the UCS Platform Emulator using KVM with virtual cloud on cloud-admin network
```
Comment (vuntz) [inline crowbar_framework/public/docs/cisco_ucs.md]: Can you wrap the lines at the same width as the rest of the document? (I guess it's 80 or 72)

Comment (vuntz) [inline crowbar_framework/public/docs/cisco_ucs.md]: Also, maybe make this line a real sentence: "Here are the steps to... on cloud-admin network:"

File: crowbar_framework/public/docs/cisco_ucs.md [hunk 2]
```diff
@@ -75,10 +75,19 @@ All troubleshooting should be done within the Cisco UCS Manager interface.
...
+  `tar xvf UCSPE_$Version.ova`
+  This results in 3 files: UCSPE_$Version-disk1.vmdk, UCSPE_$Version.ovf UCSPE_$Version.mf
+* Convert the VMDK image to qcow2 (Because image created by VMDK version 3 is ReadOnly)
+  `qemu-convert -O qcow2 UCSPE_Version-disk1.vmdk UCSPE_$Version-disk1.qcow2`
+* Create a virtual disk with at least 20GB
+* Start the VM using ucs_vm1.xml file after updating the correct Image paths.
+
+This barclamp has been tested with version 3.0(2cPE1) of the emulator.
 
 The Emulator was designed to run as a VMware virtual machine.
```
Comment (vuntz) [inline crowbar_framework/public/docs/cisco_ucs.md]: It feels like this paragraph should be the second paragraph (so before your changes).

Comment (mmnelemane) [inline crowbar_framework/public/docs/cisco_ucs.md]: @vuntz: agree. Just modified to look better.

**Comment by vuntz** [comment]
Thanks for the quick fixes, +1

**Comment by aplanas** [comment]
+1

**Comment by mmnelemane** [comment]
@aplanas: also tested on Leap.

**Comment by aplanas** [comment]
@mmnelemane Oh great, in this case I will update the doc, and there reference openSUSE 13.1, I will reference Leap 42.1 (L#82)

**Comment by mmnelemane** [comment]
@aplanas: Done.

**Comment by aplanas** [comment]
still +1 for me

**Comment by vuntz** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Pull request merged**